backend/pyproject.toml

```diff
@@ -15,15 +15,15 @@ requires-poetry = ">=2.0"
 
 [tool.poetry.dependencies]
 Django = "5.1.8"
-django-auditlog = "3.0.0"
+django-auditlog = "^3.1.2"
```

Comment:
Don't use "^" version, pin the exact version instead.

Reply:
Note to self: Don't use `poetry add ...@latest`, instead just edit the `pyproject.toml` file 😅